apps/wallet-mobile/src/features/Swap/common/SelectPool/ExpendableCard/ExpandableInfoCard.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import {Pressable, StyleSheet, Text, View} from 'react-native'
+import {TouchableOpacity} from 'react-native-gesture-handler'
+
+import {Icon, Spacer} from '../../../../../components'
+import {COLORS} from '../../../../../theme'
+
+type ExpandableInfoCardProps = {
+  label: string | React.ReactNode | null
+  mainInfo: Array<{label: string; value?: string}>
+  hiddenInfo: Array<{label: string; value: string}>
+  navigateTo?: () => void
+  buttonAction?: () => void
+  buttonText?: string
+}
+
+export const ExpandableInfoCard = ({
+  label,
+  mainInfo,
+  hiddenInfo,
+  navigateTo,
+  buttonText,
+  buttonAction,
+}: ExpandableInfoCardProps) => {
+  const [showHiddenInfo, setShowHiddenInfo] = React.useState(false)
+  return (
+    <View style={[styles.container]}>
+      <View style={styles.flexBetween}>
+        <TouchableOpacity onPress={() => navigateTo && navigateTo()}>
```

Comment:
```suggestion
        <TouchableOpacity onPress={() => navigateTo?.()}>
```

Reply:
if the desired state is to don't have the padding effect while taping on the component, onPress must be null/undefined otherwise even doing nothing touchable will animate. --> `handleOnNavigate = navigateTo ?? null`